src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -4,24 +4,30 @@
 
 public class FigureSupplier {
     public Figure getRandomFigure() {
-        Random rnd = new Random();
+        final int figureCount = 5;
+        final int Quasy = 41;
+        final int defaultSide = 10;
```

Comment:
it isn't a constant, please about it here https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names